index.html

```diff
@@ -4,42 +4,210 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css">
     <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+    <link rel="stylesheet" href="styles.css">
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
-    <title>Portfolio</title>
+    <title>Document</title>
 </head>
 <body>
-  <section class="first-page">
-    <div class="main">
-      <nav class="navbar">
-        <div class="logo"><a href="#">Eros</a></div>
-        <div class="menu"><i class='bx bx-menu'></i></box-icon></div>
-    </nav>
-    <div class="headline">
-    <div class="description">
-      <h1 class="name"> I’m Aime
-          Glad to see you!</h1>
-         
-      <p class="text"> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-           </p>
-      </div>
-      <div class="contact">
-          <h1 class="media">LET'S CONNECT:</h1>
-        <ul>
-         
-          <li><i class='bx bxl-twitter'></i></li>
-          <li><i class='bx bxl-linkedin'></i></li>
-          <li><i class='bx bxl-gmail'></i></li>
-          <li><i class='bx bxl-github'></i></li>
```

Comment:
- [ ] Hi @Hazard10-eng , this milestone of the portfolio project is a continuation of the previous project, and in each milestone of the project, after you have completed the project, you merge into the main branch, and you use it for the next project. The same here. Please undo those removed codes